quinn-proto/src/config/transport.rs

```diff
@@ -203,6 +204,12 @@ impl TransportConfig {
         self
     }
 
+    /// Specifies if all application data packets should be padded to current maximum UDP payload size.
```

Comment:
Let's add a few words explaining when this might/might not be desirable. Also, nit: no trailing period in the first line.